### DISCUSSION TITLE
Why Dovecot cannot sort threads?

### DISCUSSION THREAD

**Discussion by bilogic** [the-djmaze/snappymail#1795] [Q&A]
![image](https://github.com/user-attachments/assets/c17805c5-a27c-44a7-8561-5fe9051e274b)

I'm using dovecot. When viewing a thread, sort becomes disabled. 

Is this normal?
Which IMAP server can support this? I want to show my latest email at the top.

**Comment by fatima794** [comment]
Dovecot's threading support can be a bit tricky, especially when it comes to sorting threads in mail Here are some common reasons why it might not be sorting threads as expected:

IMAP vs. Local Mail: Dovecot is primarily an IMAP server. If your email client is set up to sort emails locally rather than relying on the server, you might not see the threading reflected in your client.

Threading Headers: For proper threading, emails need to have the appropriate headers (In-Reply-To and References). If these headers are missing or incorrectly formatted, Dovecot won't be able to thread them.

Configuration Issues: Check your Dovecot configuration for any settings related to threading or sorting. Make sure you’re using the correct backend that supports threading.

Client Limitations: Some email clients have limited support for threading or might require specific settings to enable it.

Version Compatibility: Ensure that you’re using a version of Dovecot that supports the threading features you expect.

If you need further assistance, providing more details about your setup (Dovecot version, email client, etc.) could help pinpoint the [issue!](url)https://topfollow-apk.com/

**Comment by bilogic** [comment]
I have deem the above by fatima794 to be ai-spam, could someone please delete it?
